clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/AbandonAndDecrementGoogleServerGroupAtomicOperation.groovy

```diff
@@ -67,12 +67,23 @@ class AbandonAndDecrementGoogleServerGroupAtomicOperation implements AtomicOpera
     def region = description.region
     def serverGroupName = description.serverGroupName
     def serverGroup = GCEUtil.queryServerGroup(googleClusterProvider, accountName, region, serverGroupName)
+    def isRegional = serverGroup.regional
+    // Will return null if this is a zonal server group.
```

Comment:
nit: move this comment to either the same line or the line before the `isRegional` declaration. At its current place, it makes me thing you're talking about the `def zone` line, which, if that's the case, the comment doesn't make sense.